src/main/scala/org/broadinstitute/hail/methods/LogisticRegression.scala

```diff
@@ -0,0 +1,91 @@
+package org.broadinstitute.hail.methods
+
+import breeze.linalg._
+import org.apache.spark.rdd.RDD
+import org.broadinstitute.hail.Utils._
+import org.broadinstitute.hail.annotations.Annotation
+import org.broadinstitute.hail.stats.LogisticRegressionModel
+import org.broadinstitute.hail.variant._
+import org.broadinstitute.hail.expr._
+
+object LogisticRegression {
+  def name = "LogisticRegression"
+
+  def buildX(gs: Iterable[Genotype], C: DenseMatrix[Double]): Option[DenseMatrix[Double]] = {
```

Comment:
Changed name to buildGtColumn and returning a DenseMatrix for concatenation in caller.